.github/actions/setup-rust/CHANGELOG.md

```
@@ -1,22 +1,42 @@
-## v1.0.2
+## v1.0.2 - 2025-06-20
 
 - Document `BUILD_PROFILE` environment variable and fix README example
 
-## v1.0.1
+## v1.0.1 - 2025-06-20
 
 - Document caching requirements, limitations and clarify when caches are saved.
 
-## v1.0.0
+## v1.0.0 - 2025-06-20
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)